.github/workflows/link-check.yml

```diff
@@ -0,0 +1,71 @@
+name: Link Checker
+
+on:
+  repository_dispatch:
+  workflow_dispatch:    
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
```

Comment:
Are these inputs needed? I think we can also remove repository dispatch

Reply:
no, will remove it.